npm/private/npm_package.bzl

```diff
@@ -216,6 +213,11 @@ def npm_package(
     using the `include_sources`, `include_transitive_sources`, `include_declarations`, `include_transitive_declarations`
     attributes.
 
+    The two `include*_declarations` options may cause type-check actions to run, which slows down your
```

Comment:
It's going to be a challenge for users to discover this feature... we could use `--compilation_mode fastbuild` to signal that we want to skip these typechecks, but it's less explicitly named and still not discoverable.

Reply:
We think we can instrument both `js_run_devserver` and `js_test` such that we can detect the value of the flag, and warn "it looks like you're doing a development loop and still have extra type-checks turned on"

Reply:
will try this in a followup. #1206